sql/src/main/java/org/apache/druid/sql/calcite/planner/CalciteRulesManager.java

```diff
@@ -233,7 +233,13 @@ public List<Program> programs(final PlannerContext plannerContext)
 
     boolean isDebug = plannerContext.queryContext().isDebug();
     return ImmutableList.of(
-        Programs.sequence(preProgram, Programs.ofRules(druidConventionRuleSet(plannerContext))),
+        Programs.sequence(
+              new LoggingProgram("Start", isDebug),
+              preProgram,
+              new LoggingProgram("After preProgram", isDebug),
+              Programs.ofRules(druidConventionRuleSet(plannerContext)),
+              new LoggingProgram("After volcano planner program", isDebug)
+            ),
```

Comment:
I'm guessing you added this for debugging, I suspect you intended to remove before merge.